modules/gapi/test/cpu/gapi_core_tests_cpu.cpp

```diff
@@ -369,11 +369,14 @@ INSTANTIATE_TEST_CASE_P(LUTTestCustomCPU, LUTTest,
                                 Values(cv::compile_args(CORE_CPU))));
 
 INSTANTIATE_TEST_CASE_P(ConvertToCPU, ConvertToTest,
-                        Combine(Values( CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1 ),
+                        Combine(Values(CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1),
                                 Values(CV_8U, CV_16U, CV_16S, CV_32F),
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
+                                Values(3.0, 1.0, 0.5, 0.0, -1.0),
+                                Values(256.0, 32.0, 0.0, -128.0),
+                                Values(Tolerance_FloatRel_IntAbs(1e-5, 2).to_compare_f()),
```

Comment:
I believe there should be `AbsExact` comparison function (and for GPU as well)

Reply:
Done, thanks!